cubic-server/world-storage/ChunkColumn.hpp

```diff
@@ -0,0 +1,83 @@
+#include <cstdint>
+#include <array>
+#include <vector>
+#include <deque>
+
+// #include "Entity.hpp"
+#include "protocol/Structures.hpp"
+
+#ifndef CHUNKCOLUMN_HPP
+#define CHUNKCOLUMN_HPP
+
+constexpr int CHUNK_2D_SIZE = 16*16;
+constexpr int CHUNK_3D_SIZE = CHUNK_2D_SIZE*16;
+constexpr int BIOME_3D_SIZE = 4*4*4;
+constexpr int NB_OF_CHUNKS = 20;
+
+struct block { // TODO: PTDR
+};
+
+struct block_entity {
+};
+
+struct height_map {
```

Comment:
Can you name all the structs in PascalCase like the classes please ?

Reply:
👍